README.md

```diff
@@ -13,44 +13,7 @@ https://github.com/britzl/defold-sharing/archive/master.zip
 
 Or point to the ZIP file of a [specific release](https://github.com/britzl/defold-sharing/releases).
 
-
-Starting Defold 1.2.162 this extension has one more dependency:
-
-https://github.com/defold/extension-gps/archive/master.zip
-
-
-<details><summary>You don't need to merge manifests manually starting Defold 1.2.153</summary>
-<p>
-	
-## Additional steps for iOS
```

Comment:
Don't we still need the instructions for iOS?

Reply:
Isn't it the same for Android? These settings are all added automatically with the manifest merging?

See `share/manifests/android/AndroidManifest.xml` and `share/manifests/ios/Info.plist`

Reply:
Ah yes, you're right!